README.md

```diff
@@ -21,11 +23,15 @@ NCL to Python consists of pages that explain specifically how to achieve somethi
 in NCL in Python, including any algorithmic differences, guidance regarding replication under different
 conditions or circumstances, and any other relevant comparisons between the NCL and Python functionality
 
+This is part of the GeoCAT's commitment to NCL and NSF NCAR's [Pivot to Python Initiative](https://www.ncl.ucar.edu/Document/Pivot_to_Python/)
```

Comment:
```suggestion
This is part of the GeoCAT team's commitment to NCL and NSF NCAR's [Pivot to Python](https://www.ncl.ucar.edu/Document/Pivot_to_Python/)
```
I think this is a little more clear / correct. 

It'd be good to get @erogluorhan's eyes on this as well to make sure this is worded in a way he's comfortable with (re: the GeoCAT team commitment).

Reply:
This reworded one sounds good to me even though I'd find the previous one also fine. Thanks!